unit_test/CMakeLists.txt

```diff
@@ -106,7 +106,7 @@ list(APPEND FWK_SRC ${UNITY_SRC_ROOT}/unity.c)
 list(APPEND SCP_UNITY_SRC ${TEST_ROOT}/unity_mocks/scp_unity.c)
 
 #Append common unit tests below here
-list(APPEND UNIT_MODULE dvfs)
```

Comment:
Please don't remove dvfs unit tests
list(APPEND UNIT_MODULE atu)
list(APPEND UNIT_MODULE dvfs)

Reply:
Sorry, I missed this. Thanks for pointing out.